packages/fxa-admin-panel/package.json

```diff
@@ -23,6 +23,7 @@
     "dateformat": "^3.0.3",
     "express": "^4.17.1",
     "express-http-proxy": "^1.6.0",
+    "fxa-components": "file:../fxa-components",
```

Comment:
these new `file:` dependencies inform lerna about the inter-package dependencies so that it can run commands (like build) in a logical order